tools/handle-ddl-lock.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Handling the Sharding DDL Lock Manually
+summary: Learn to manually unlock the sharding DDL lock.
```

Comment:
Should we call it "Troubleshooting Sharding DDL Locks"?  'Handling' is a bit ambiguous: it could be handling large amounts of data etc.  This sounds like break/fix? (troubleshooting)